### PR TITLE
Landing page content

### PR DESCRIPTION
**Pull request by abziegler**
For landing page, adding faculty images, activating CTAs, correcting typo

### CODE REVIEW

File: app/views/base/index.html.erb [hunk 1]
```diff
@@ -74,20 +72,12 @@
...
-</section>
 <section class="stinger">
   <%= image_tag 'logo.png', title: t('header.brand') %>
   <h1>
     <%= t 'landing.stinger.pitch' %>
   </h1>
-  <a class="call-to-action">
+  <a class="call-to-action" href="<%= new_user_path %>">
     <%= t 'landing.stinger.call-to-action' %>
   </a>
```
Comment (cgruppioni) [inline app/views/base/index.html.erb]: <%= link_to  t('landing.stinger.call-to-action'), new_user_path, class: "call-to-action" %>

Comment (cgruppioni) [inline app/views/base/index.html.erb]: You can replace lines 80-82 with
```<%= link_to t('landing.stinger.call-to-action'), new_user_path, class: "call-to-action" %>```

File: app/views/base/index.html.erb [hunk 2]
```diff
@@ -28,9 +26,9 @@
           </section>
         <% end %>
         <section>
-          <a class="call-to-action" href="<%= root_path %>">
-            <%= t('landing.preview.call-to-action') %>
-          </a>
+          <div class="link">
```
Comment (cgruppioni) [inline app/views/base/index.html.erb]: Is this class necessary? I don't see it being used for styling but I could be missing something

File: app/views/base/index.html.erb [hunk 3]
```diff
@@ -28,9 +26,9 @@
...
+            <%= link_to t('footer.links.user-guide'), page_path('docs/guide'), :class=>"call-to-action" %>
```
Comment (cgruppioni) [inline app/views/base/index.html.erb]: Not important but the "new" ruby syntax is 
```class: "call-to-action"```
it renders the same as 
```:class => "call-to-action"```

File: app/assets/stylesheets/landing.scss [hunk 4]
```diff
@@ -74,6 +74,9 @@ section.preview {
 section.endorsements {
   text-align: center;
 
+  border: 4px solid $light-gray;
+  border-width: 4px 0 4px;
```
Comment (cgruppioni) [inline app/assets/stylesheets/landing.scss]: You can just do ```border-width: 4px 0;```

**Comment by abziegler** [comment]
Thanks for the review. I fixed the things you requested and also added two more faculty.

**Pull request merged**